src/render-html/css.js

```diff
@@ -277,6 +277,10 @@ ul {
   margin: 0.25em 0;
   font-size: 10px;
 }
+div.twitter-image > a > img {
+  height: auto;
+  max-width: 150px;
```

Comment:
We need to restrict the height not the width. There is nothing bad about wide image.